api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -127,43 +138,46 @@
     @Value("${api.search.hl.MaxAnalyzedChars}")
     private String hlMaxAnalyzedChars;
 
+    public static void main(String args[]) {
+        String[] profiles = {"amit", "rahul", "surya"};
+        System.out.println(StringUtils.join(profiles, ","));
+    }
+
     /**
      * Returns a list of Europeana datasets based on the search terms.
      * The response is an Array of JSON objects, each one containing the identifier and the name of a dataset.
      *
      * @return the JSON response
      */
     @ApiOperation(value = "search for records post", nickname = "searchRecordsPost", response = Void.class)
-    @PostMapping(value = {
-            "/api/v2/search.json",
-            "/record/v2/search.json",
-            "/record/search.json"
-    }, produces = MediaType.APPLICATION_JSON_VALUE, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public ModelAndView searchJsonPost(
-            @RequestParam(value = "wskey") String apikey,
-            @RequestBody SearchRequest searchRequest,
-            HttpServletRequest request,
-            HttpServletResponse response) throws EuropeanaException {
+    @PostMapping(value = {"/api/v2/search.json", "/v2/search.json", "/record/v2/search.json", "/record/search.json"},
```

Comment:
"/v2/search.json" is not a proper endpoint we want to advertise

Reply:
same puzzlement here: I did not change any of that (at least not consciously so).